templater/templates/periodics.yaml

```
@@ -58,6 +58,7 @@ periodics:
       - -c
       - >
         {{- if .imageBuild }}
+        trap 'touch /status/done' EXIT &&
```

nit: can we move the `&&` to the next line for consistency?
```suggestion
        trap 'touch /status/done' EXIT
        &&
```

---

done